src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -186,13 +187,19 @@ export enum MutationTableColumnType {
     GENE_PANEL = 'Gene panel',
     SIGNAL = 'SIGNAL',
 }
-type ExtendedMutationTableColumnType = MutationTableColumnType | string;
+export type ExtendedMutationTableColumnType = MutationTableColumnType | string;
 
-type MutationTableColumn = Column<Mutation[]> & {
+export type MutationTableColumn = Column<Mutation[]> & {
     order?: number;
     shouldExclude?: () => boolean;
 };
 
+export type NamespaceColumnConfig = {
+    [namespaceName: string]: {
```

Comment:
I'm not sure if this is the right place for it, but a comment explaining what a namespace column is would be good

Reply:
I added come comment lines to explain this feature to frontend-devs.

Reply:
great, thank you.